lean4-infoview-api/src/infoviewApi.ts

```diff
@@ -128,6 +128,11 @@ export interface InfoviewApi {
    */
   requestedAction(action: InfoviewAction): Promise<void>;
 
+  /**
+   * Interface for running test scripts inside the infoview webview.
```

Comment:
```suggestion
   * Execute the given JavaScript code inside the infoview. Must not be used
   * for anything other than testing.
```
To clarify that the argument is source code rather than e.g. the name of a test script.